frontend/components/FeatureList.vue

```diff
@@ -0,0 +1,280 @@
+                        {
+                            text: `The tool redirects the user to indicated services or github repos of indicated services`,
+                            indent: 0,
+                        },
+                    ],
+                },
+                {
+                    title: `Collateral auctions - dashboard page`,
+                    url: `https://auctions.makerdao.network/dashboard`,
+                    features: [
+                        {
+                            text: `The tools shows a list of supported collateral types with auction related parameters`,
+                            indent: 0,
+                        },
+                        {
+                            text: `Uniswap price`,
+                            indent: 1,
+                        },
+                        {
+                            text: `step parameter`,
+                            indent: 1,
+                        },
+                        {
+                            text: `cut parameter`,
+                            indent: 1,
+                        },
+                        {
+                            text: `Token icon`,
+                            indent: 1,
+                        },
+                        {
+                            text: `Link to token contract on etherscan`,
+                            indent: 1,
+                        },
+                    ],
+                },
+                {
+                    title: `Twitter Bot`,
+                    url: `https://twitter.com/MakerDAO_SAS`,
+                    features: [
+                        {
+                            text: `The bot informs via a tweet about each new started collateral auction`,
+                            indent: 0,
+                        },
+                        {
+                            text: `The message indicates the amount of collateral that is auctioned off`,
+                            indent: 1,
+                        },
+                        {
+                            text: `The message includes a link to the respective auction in the tool (ie. 
+                            Collateral Auction UI)`,
+                            indent: 1,
+                        },
+                    ],
+                },
+                {
+                    title: 'Auction Keeper',
+                    url: undefined,
+                    features: [
+                        {
+                            text: `The auction keeper is an open source tool provided by us for download`,
+                            indent: 0,
+                        },
+                        {
+                            text: `The keeper is not operated by us for or on behalf of the user`,
+                            indent: 0,
+                        },
+                        {
+                            text: `A user is able to download the code and run it himself`,
```

Comment:
This is more a comment for @florianstoecker. 

I believe we should be using gender neutral language. Therefore I would replace `himself` with `themself`.

Reply:
Had a quick talk with Flo about this. He agrees it is good to change this immediately!

Reply:
Very nice catch!